### PR TITLE
Rename category to catalogue, add setup.py and is_public

### PR DESCRIPTION
**Pull request by viggo-devries**

### CODE REVIEW

File: setup.py
Comment (timsavage) [inline setup.py:1]: Please remove this file. This package is using poetry for dependency management building of wheel files.

**Review by timsavage — CHANGES_REQUESTED** [review]
Tests also need to be updated to refer to catalogue.

File: setup.py
Comment (timsavage) [reply setup.py:1]: `poetry build` will generate a wheel and `.tar.gz` source package in the `dist` folder.

**Comment by viggo-devries** [comment]
@timsavage Please check again.

**Pull request closed without merging**